app/src/androidTest/java/com/github/wnder/NewPictureTesting.java

```diff
@@ -42,6 +45,22 @@ public static void createPic() throws InterruptedException {
         }
     }
 
+    // Clean db
+    @AfterClass
+    public static void deleteTestPic() {
+        try {
+            sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        FirebaseFirestore.getInstance().collection("pictures").document(pic.getUniqueId())
+                .collection("userData").document("userGuesses").delete();
+        FirebaseFirestore.getInstance().collection("pictures").document(pic.getUniqueId())
+                .collection("userData").document("userScores").delete();
+        FirebaseFirestore.getInstance().collection("pictures").document(pic.getUniqueId()).delete();
+        FirebaseStorage.getInstance().getReference().child("pictures/"+pic.getUniqueId()+".jpg").delete();
```

Comment:
Maybe creating a method in Storagewould be better

Reply:
As we don't have any feature to delete a picture in the app I don't think it would make sense to have a method there to do so.
We might do it later but I don't think it's part of any user stories yet.

Reply:
Alright then